container/Dockerfile

```diff
@@ -112,6 +113,8 @@ RUN mkdir /var/www/.composer && chown -R www-data:www-data /var/www/.composer ;
     sudo -u www-data cp -a /var/www/MISP/app/Config/core.default.php /var/www/MISP/app/Config/core.php ; \
     sudo -u www-data cp -a /var/www/MISP/app/Config/config.default.php /var/www/MISP/app/Config/config.php
 
+VOLUME /var/www/MISP/app/Config
```

Comment:
I had to move this from where it was previously because otherwise the git clone into the /var/www/MISP directory earlier would fail because the directory was non-empty. Maybe there's a better way to do this though?

Reply:
The only problem here is that per Docker the way volume mounts work is that if you make any changes after declaring them - those changes are lost.

Specifically:
```
"if any build steps change the data within the volume after it has been declared, those changes will be discarded."
```

This basically summarizes as having to declare Volumes "up above" before anything actually utilizes them/makes changes to them.

Reply:
@jbarlow-mcafee ^ Just pinging about this in case it was lost among the emails from the pull request/merges/etc.

Reply:
@ventz Ah, sorry. I apparently missed the e-mail notifications for your responses. I agree that the order for the volume mounts would be a problem with the changes as I have them here. I've been working on a separate idea for how these volumes could be mounted by default. As these changes would be more extensive than what I have here, I'll put up a separate PR with those and just close this one out for now. Thanks again!

Reply:
I put up #17 as a replacement for this PR.